docs/pages/components/step-input.md

```diff
@@ -0,0 +1,181 @@
+---
+title: Step Input
+id: step-input
+keywords: Step Input, Step, Input
+sidebar: left-navigation-sidebar
+toc: false
+permalink: components/step-input.html
+folder: components
+---
+
+The Step Input component is an opinionated composition of the Input, Button components.
+It is typically used to enter numbers.The value can be increased or reduced with the provided controls.
+{: .docs-intro}
+
+## Step Input with compact
+
+{% capture step-input-compact %}
+
+<div class="fd-step-input">
```

Comment:
Playing around with the branch, I'm failing to see why `<div class="fd-step-input">` is necessary. Can you explain why?